### PR TITLE
ipython 4.0.0b1 does not work (yet)

### PR DESCRIPTION
**Pull request by motivator**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 distribute
 argparse
 sqlalchemy>=0.7
-IPython>=0.10
+IPython>=0.10,<4.0.0b1
```
Comment (thekad) [inline requirements.txt]: change to <4.0 instead?

Comment (motivator) [inline requirements.txt]: doesn't work. <4, <4.0, <4.0.0 don't work because technically 4.0.0b1 is less than all of those i guess :(

**Comment by thekad** [comment]
lolsob :+1:

**Comment by JTCunning** [comment]
Schwomp :+1:

**Pull request merged**